README.md

```diff
@@ -2,62 +2,79 @@
 
 [![link checks](https://github.com/saturncloud/examples/workflows/link%20checks/badge.svg?branch=main)](https://github.com/saturncloud/examples/actions/workflows/check-links.yml)
 
-These are the Saturn Cloud Default Resources that populate admin accounts and cannot be deleted. They are used to power template resources (which can turned on or off by Saturn Cloud admins). Each example has a set of files to include on workspace machine when the resource is started, as well as parameters for how the resource should be set up. These parameters match those which users set when creating new resources. If you would like to use code from one of these resources, select one of the template resources from the resource page of Saturn Cloud.
+These are the Saturn Cloud Examples most of which are used to create default resource templates on every Saturn Cloud installation (this behavior can be turned on or off by Saturn Cloud admins).
+
+Each example has a [recipe](https://github.com/saturncloud/recipes) that describes how the resource should be setup as well as files (notebooks, markdown, python scripts) that contain the actual example content.
+If you would like to explore these examples, select one of the template resources from the resources page of Saturn Cloud.
```

Comment:
Since not all of these are template resources, I was going to suggest we put links in the readme of each example to create a resource, however that would create some weird self-referrential stuff if you read the readme from within the resource. So how about:

```suggestion
If you would like to explore these examples, select one of the template resources from the resources page of Saturn Cloud or view the examples in the [Saturn Cloud docs](https://saturncloud.io/docs/examples).
```

Reply:
Oh I think we should 100% have buttons on the READMEs within the examples. I just didn't put that bit yet since it isn't quite ready